Support docker system config

LmP maintains a system docker configuration which should also be added
to the default config search path.

Upstream-Status: Inappropriate [lmp specific]

diff --git a/src/import/vendor/github.com/containers/image/v5/pkg/docker/config/config.go b/src/import/vendor/github.com/containers/image/v5/pkg/docker/config/config.go
index cf82ee86..2fcad4f7 100644
--- a/src/import/vendor/github.com/containers/image/v5/pkg/docker/config/config.go
+++ b/src/import/vendor/github.com/containers/image/v5/pkg/docker/config/config.go
@@ -37,6 +37,7 @@ var (
 	defaultPerUIDPathFormat = filepath.FromSlash("/run/containers/%d/auth.json")
 	xdgConfigHomePath       = filepath.FromSlash("containers/auth.json")
 	xdgRuntimeDirPath       = filepath.FromSlash("containers/auth.json")
+	dockerSystemConfigPath  = filepath.FromSlash("/usr/lib/docker/config.json")
 	dockerHomePath          = filepath.FromSlash(".docker/config.json")
 	dockerLegacyHomePath    = ".dockercfg"
 	nonLinuxAuthFilePath    = filepath.FromSlash(".config/containers/auth.json")
@@ -142,6 +143,8 @@ func getAuthFilePaths(sys *types.SystemContext) []authPath {
 		xdgCfgHome = filepath.Join(homedir.Get(), ".config")
 	}
 	paths = append(paths, authPath{path: filepath.Join(xdgCfgHome, xdgConfigHomePath), legacyFormat: false})
+	// LmP: Load values from system config by default
+	paths = append(paths, authPath{path: dockerSystemConfigPath, legacyFormat: false})
 	if dockerConfig := os.Getenv("DOCKER_CONFIG"); dockerConfig != "" {
 		paths = append(paths,
 			authPath{path: filepath.Join(dockerConfig, "config.json"), legacyFormat: false},
